src/guides/v2.3/install-gde/prereq/security.md

```diff
@@ -48,7 +48,7 @@ The preceding commands work only with the Apache web server. Because of the vari
 
 ### Enable inter-server communication
 
-If Apache and the database server are on the same host, you can skip this section and continue with [Opening Ports In Your Firewall](#install-iptables).
+If Apache and the database server are on the same host, you will still need to use the following command if you are planning to use integrations that use curl (ex. Paypal and USPS).
```

Comment:
I would say 

```suggestion
If Apache and the database server are on the same host, you may still need to use the following command if you are planning to use integrations that use curl (ex. Paypal and USPS).
```

The reason is, it's not basically Apache that uses curl. PHP does not require Apache to run curl commands. But with the more or less standard apache installation, the curl command will be executed via the Apache user/group. If your installation has different user/group/security settings you don't need to enable the SELinux command as described. 

Thank you